jenkins/scripts/deploy-dev.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+
+#
+# Will change this so it copies the latest build created by Jenkins
+# so the DEV server won't have to rebuild an already built project.
+#
+
+ORG_PATH="~/JavaPlebes"
+PROJ_DIR="/TextAdventure"
+GIT_URL="git@github.com:javaplebes/TextAdventure.git"
+
+if [ ! -d "$ORG_PATH" ]; then
+  echo "Organisation path does not exist. Creating..."
+  mkdir ORG_PATH
+fi
+
+cd $ORG_PATH
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2164
Lets use something like `cd $ORG_PATH || { echo "Unable to change directory to $ORG_PATH"; exit 1; }`
What do you think?

Reply:
Just to be on the safer side. Since cd can fail for a lot of reasons apart from missing dir (which we are already handling in the script).